lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_shape_inference.cpp

```diff
@@ -104,15 +102,15 @@ torch::lazy::Shape InferStack(const ir::ops::Stack* stack) {
   return torch::lazy::Shape(input_shape.scalar_type(), output_dimensions);
 }
 torch::lazy::Shape InferShape(const torch::lazy::Node* node) {
-  if (node->op() == *ir::ops::ltc_generic_slice) {
+  if (node->op() == ir::ops::generic_slice) {
     auto generic_slice = torch::lazy::NodeCast<ir::ops::GenericSlice>(
-        node, *ir::ops::ltc_generic_slice);
+        node, ir::ops::generic_slice);
     const torch::lazy::Output& argument = node->operand(0);
     return torch::lazy::Shape(
         torch::lazy::GetShapeFromTsOutput(argument).scalar_type(),
         generic_slice->sizes());
   }
-  if (node->op() == *ir::ops::ltc_update_slice) {
+  if (node->op() == ir::ops::generic_slice_reverse) {
```

Comment:
what's up with dropping the * operator here?

Reply:
I changed the declaration of this op into plain OpKind (see opcode.h) instead of OpKindWrapper. I don't see a value   of using OpKindWrapper.